ofborg/test-srcs/build/default.nix

```
@@ -1,27 +1,27 @@
 let
-  nix = import <nix/config.nix>;
-in {
+  builder = builtins.storePath <ofborg-test-bash>;
+in
+
+{
   success = derivation {
     name = "success";
     system = builtins.currentSystem;
-    builder = builtins.storePath nix.shell;
-    args = [
-      "-c"
-      "echo hi; echo ${toString builtins.currentTime} > $out" ];
+    inherit builder;
+    args = ["-c" "echo hi; echo ${toString builtins.currentTime} > $out" ];
```

Nit (applies to all `args = [` in these files):

```suggestion
    args = [ "-c" "echo hi; echo ${toString builtins.currentTime} > $out" ];
```

or

```suggestion
    args = ["-c" "echo hi; echo ${toString builtins.currentTime} > $out"];
```

(matching spaces -- or lack thereof -- on both ends)